src/c++/devices/tracer/dmTracer.c

```diff
@@ -451,8 +451,8 @@ static struct target_type tracerTargetType = {
 /**********************************************************************/
 int __init tracerInit(void)
 {
-  STATIC_ASSERT(offsetof(TracerDevice, dev)
-                == offsetof(CommonDevice, dev));
+  BUILD_BUG_ON(offsetof(TracerDevice, dev)
+               != offsetof(CommonDevice, dev));
```

Comment:
If you're going to unwrap the line, I  think you should unwrap the line in all five test devices.